src/Integration.Vsix.UnitTests/CFamily/CFamilyIssueToAnalysisIssueConverterTests.cs

```diff
@@ -369,9 +369,6 @@ public void ConvertFromIssueSeverity_InvalidValue_Throws()
         public void ConvertFromIssueType(IssueType cfamilyIssueType, AnalysisIssueType analysisIssueType)
         {
             CFamilyIssueToAnalysisIssueConverter.Convert(cfamilyIssueType).Should().Be(analysisIssueType);
-
```

Comment:
Misc: noticed this when blatantly copying the tests; it's a redundant assertion that belongs to the test below.